client/src/javascript/components/modals/add-torrents-modal/AddTorrentsByURL.js

```diff
@@ -17,14 +18,15 @@ class AddTorrentsByURL extends React.Component {
     errors: {},
     isAddingTorrents: false,
     tags: '',
-    urlTextboxes: [{value: ''}],
+    urlTextboxes: UIStore.getActiveModal().torrents || [{id: 0, value: ''}],
```

Comment:
This breaks if `getActiveModal()` returns `null`

Reply:
I have now added a check for this.